config/customFormats.js

```diff
@@ -13,6 +15,51 @@ const getLeaf = (object) => {
   };
 };
 
+function MDX(dictionary) {
+  const topLevels = Object.entries(dictionary.properties);
+  let str = ``;
+
+  function MDXContent(dictionary, level, first) {
+    const category = dictionary[0];
+
+    let isToken = dictionary[1].value;
```

Comment:
This check will fail for cases where the value is set but falsey (0, "", null, undefined). We might want to do this instead: 
```suggestion
    let isToken = dictionary[1].hasOwnProperty("value");
```

Reply:
Good catch, thanks!

Reply:
Sorry for the nit, was mostly just excited to use the "new" github suggestion feature 😂